module/TaskManagement/src/TaskManagement/Controller/TasksController.php

```diff
@@ -238,6 +246,43 @@ public function delete($id)
         return $this->response;
     }
     
+	public function applytimeboxforsharesAction(){

```

Comment:
Il nome dell'azione non ne riflette la semantica.
Non è chiaro come è stata modellata l'operazione in un'ottica RESTful